cisco_umbrella_reporting/help.md

```diff
@@ -0,0 +1,124 @@
+# Description
+
+This plugin utilizes Cisco Umbrella to get the most complete view of the relationships and evolution of Internet domains, IP addresses, and autonomous systems to pinpoint attackers infrastructures and predict future threats
+
+# Key Features
+
+* Most Recent Requests
+* Security Activity Report
+
+# Requirements
+
+* Requires an API Key and Secret from the Cisco Umbrella
+* Requires organization ID
```

Comment:
```suggestion
* Requires organization ID
```
Which organization ID is this referring to? Cisco Umbrella? This should be specified, as a user may confuse it with a Rapid7 organization ID.